release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -5229,3 +5229,23 @@ $ oc adm release info 4.12.78 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.12.79
+[id="ocp-4-12-79_{context}"]
+=== RHSA-2025:12323 - {product-title} 4.12.79 bug fix and security update
+
+Issued: 07 August 2025
+
+{product-title} release 4.12.79 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:12323[RHSA-2025:12323] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:12324[RHBA-2025:12324] advisory.
+
```

Comment:
Add this sentence here with spaces around it (right before "You can view the container images...":

Space precluded documenting all of the container images for this release in the advisory.